product_docs/docs/epas/15/reference/sql_reference/02_data_types/01_numeric_types.mdx

```diff
@@ -5,27 +5,45 @@ legacyRedirectsGenerated:
+| `NUMERIC`           | ✅      |       | User-specified precision, exact; variable storage, up to 131072 digits before the decimal point; up to 16383 digits after the decimal point range                                                                                   |
+| `NUMBER`            |        | ✅     | Alias for native numeric                                                                                                                                                                                                            |
+| `NUMBER(p [, s ] )` |        | ✅     | Alias of native numeric with exact numeric of maximum precision, `p`, and optional scale, `s`; variable storage, p to 1000 digits of precision                                                                                      |
+| `REAL`              | ✅      |       | Variable-precision, inexact; 4 bytes	storage;	6 decimal digits precision range                                                                                                                                                        |
+| `DOUBLE PRECISION`  | ✅      |       | Variable-precision, inexact; 8 bytes storage; 15 decimal digits precision range                                                                                                                                                     |
+| `BINARY FLOAT`      |        | ✅     | Alias for native `DOUBLE PRECISION`                                                                                                                                                                                                 |
+| `SMALLSERIAL`       | ✅      |       | Small autoincrementing integer,  2 bytes storage,	1 to 32767 range                                                                                                                                                                   |
+| `SERIAL`            | ✅      |       | Autoincrementing integer, 4 bytes	storage,	1 to 2147483647 range                                                                                                                                                                      |
+| `BIGSERIAL`         | ✅      |       | Large autoincrementing integer, 8 bytes	storage,	1 to 9223372036854775807 range                                                                                                                                                       |
+| `ROWID`             |        |       | Custom type for emulating Oracle ROWID, signed 8 bit integer; 8 bytes storage; -9223372036854775808 to 9223372036854775807 range (see [Migration Handbook](/migrating/oracle/oracle_epas_comparison/database_features/#data-types)) |
+
+## Overview 
+
+Numeric types consist of four-byte integers, four-byte and eight-byte floating-point numbers, and fixed-precision decimals. 
```

Comment:
@nidhibhammar,, @josh-heyer, @djw-m - PG says "selectable-precision decimals". We say "fixed-precision decimals". Is that a meaningful difference?

Reply:
Based on DJ's comments, I'll leave the text as "fixed-precision".